frontend/src/components/BestExpertiseSection/BestExpertiseSection.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {useState, useEffect} from 'react';
+import Card from './Card';
+import { API } from '../../modules/apis';
+
+
+const BestExpertiseSection = () => {
+  const [services, setServices] = useState([]);
+  useEffect(() => {
+    API('/services').then(({ data: services }) => {
+      setServices(services);
+    });
+  }, []);
+  const cards = services.map(({title, details, button}) => {
+    return <Card title={title} details={details} button={button} key={Math.random()}/>;
```

Comment:
the key should be a thing you absolutely sure it will be unique,
I suggest adding an id value to the **API** and use it.